cli/src/find_path.rs

```diff
@@ -15,19 +17,20 @@ pub fn find_path(client: &Client, matches: &ArgMatches) -> Result<()> {
         let source_account = matches
             .value_of("from")
             .expect("Source account is a required field");
-        let destination_amount: Amount = matches
+        let destination_amount = matches
             .value_of("amount")
             .expect("Destination amount is a required field")
-            .parse()
-            .expect("Amount must be properly formatted");
-        let destination_asset_str = matches
+            .parse::<Amount>()
+            .map_err(|_| String::from("Amount must be a valid i64 integer"))?;
+        let destination_asset_str: AssetIdentifier = matches
```

Comment:
This isn't a string any more after it's been parsed 😉 

you may want to remove the `_str` from the name.

Reply:
Valid point